### PR TITLE
flowTastic

### PR DESCRIPTION
**Pull request by NKTTT**
Bubble sorting with Go

### CODE REVIEW

**Comment by NKTTT** [comment]
You're an assigner now too!

File: flowTastic [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+func bubbleSort(numbers []int) []int {
+	N := len(numbers)
+	for i := 0; i < N; i++ {
+		if !sweep(numbers, i) {
+			return numbers
+		}
+	}
+	return numbers
+}
+
```
Comment (zzergAtStage) [inline flowTastic:18]: Make more readable func names - what it do certainly

**Review by zzergAtStage — APPROVED** [review]
All right

**Comment by zzergAtStage** [comment]
Close a task after mergin

**Pull request merged**